setup-build-freetype.py

```diff
@@ -42,7 +42,9 @@
                              prefix_dir, prefix_dir)
 
 # Try to use Ninja to build things if it's available. Much faster.
-if distutils.spawn.find_executable("ninja"):
+# On Windows, I first need to figure out how to make it aware of VC, bitness,
+# etc.
```

Comment:
what was the issue with ninja on windows? cmake should tell ninja everything it needs to know.
what versions of cmake/ninja are you using?

Reply:
cmake 3.10.x and the most up-to-date Ninja I could download. Last time I tried, it failed to find a compiler... maybe just a path problem, but I wanted to build a bwheel and worry about Ninja later. Does it work for you?

Reply:
If I run setup.py from within the "Visual C++ 2015 x64 Native Build Tools Command Prompt", it works

Reply:
well, actually if fails later when linking with unresolved symbols.. argh. Sorry can't debug this further.

Reply:
I want it to run from a normal Powershell without "* Command Prompt" frickery... some other time. msbuild gets the job done somehow.